tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/evaluation/ExpressionEvaluationTest.java

```diff
@@ -172,7 +172,9 @@ public void variableReferenceEvaluationTest() throws BallerinaTestException {
 
         debugTestRunner.assertExpression(context, "nameWithType", "\"Ballerina\"", "string");
         debugTestRunner.assertExpression(context, "nameWithoutType", "\"Ballerina\"", "string");
-        debugTestRunner.assertExpression(context, GLOBAL_VAR_03, "map (size = 1)", "map");
+        debugTestRunner.assertExpression(context, GLOBAL_VAR_03,
+                                         "(debug_test_resources/evaluation_tests:0:$anonType$_10 & readonly)",
```

Comment:
We should ideally be the shape. We may also not be able to rely on the number suffix being the same.